roles/vgpu/defaults/main.yml

```diff
@@ -13,5 +16,5 @@ vgpu_mig_definitions: []
 vgpu_definitions: "{{ vgpu_mig_definitions }}"
 
 # Packages providing nvidia-mig-manager
-vgpu_nvidia_mig_manager_rpm_url: https://github.com/NVIDIA/mig-parted/releases/download/v0.5.1/nvidia-mig-manager-0.5.1-1.x86_64.rpm
-vgpu_nvidia_mig_manager_deb_url: https://github.com/NVIDIA/mig-parted/releases/download/v0.5.1/nvidia-mig-manager_0.5.1-1_amd64.deb
+vgpu_nvidia_mig_manager_rpm_url: https://github.com/NVIDIA/mig-parted/releases/download/v0.12.1/nvidia-mig-manager-0.12.1-1.x86_64.rpm
```

Comment:
Would be nice to all out this change from v0.5 to v0.12 in the PR (i.e. in the first comment)

Reply:
Done. I've called out the other changes too.